test/components/form/preview.spec.js

```diff
@@ -0,0 +1,56 @@
+import testData from '../../data';
+import { load } from '../../util/http';
+import { mockLogin } from '../../util/session';
+import Modal from '../../../src/components/modal.vue';
+
+describe('FormPreview', () => {
+  // eslint-disable-next-line no-console
+  const originalConsoleLog = console.log;
+
+  beforeEach(() => {
+    mockLogin();
+    // eslint-disable-next-line no-console
+    console.log = () => {};
+  });
+
+  afterEach(() => {
+    // eslint-disable-next-line no-console
+    console.log = originalConsoleLog;
+  });
```

Comment:
Is mocking `console.log()` needed now that `useFeatureFlags()` is checking `process.env.NODE_ENV`?